development/process.md

```diff
@@ -82,10 +82,29 @@ keep the amount of rework from the contributor to a minimum.
     3. Go to step 5.
 
 8. Maintainer merges the pull request after final changes are accepted.
+
+    * Approval of one maintainer is required for merge. This includes cases when
```

Comment:
The wording on this is a little awkward, and I'm actually not sure if it's saying you need two maintainers when one is an author, or if you need one maintainer period. If the latter, maybe just "A maintainer can merge a pull request that they authored as long as it is approved by another reviewer."